app/front/scripts/components/data-package-api/index.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * Created by Ihor Borysyuk on 18.02.16.
+ */
+
+'use strict';
+var Promise = require('bluebird');
+var downloader = require('../downloader');
+var _ = require('underscore');
+
+module.exports = function (config) {
+  var _config = config;
```

Comment:
Why do you create this strange var? Since it is an object, it will not be cloned; is not changed in this function; and so on. Maybe, it is better just to use `config`